import-service/handlers/importFileParser.ts

```diff
@@ -25,6 +28,16 @@ const importFileParser = async (
                   let record = parser.read();
                   while (record) {
                     console.log(record);
+                    console.log(process.env.IMPORT_SQS);
+                    sqs.sendMessage(
```

Comment:
How do you control the amount of sendMessage requests being in flight at the same time? (in case csv file is huge and scv parser produces data fast)

Reply:
Never think about it. Does it increase costs?

Reply:
Theoretically might be some issues with the amount of callbacks being kept in the memory